packages/backend/src/managers/playground.ts

```diff
@@ -212,6 +212,7 @@ export class PlayGroundManager {
             Type: 'bind',
           },
         ],
+        SecurityOpt: [DISABLE_SELINUX_LABEL_SECURITY_OPTION],
```

Comment:
What are the implication of this security option ? Why does it is required ? I cannot really understand the link with applehv from the documentation https://docs.podman.io/en/v4.6.1/markdown/options/security-opt.html

Reply:
The volumes are not readable if this option is not set on Linux and applehv. Inspired from https://github.com/containers/podman/issues/3683

Reply:
So we might want to include this option on windows or we don't care ?

Reply:
It is set for all platforms, I prefer to have a single general flow